sensu/plugins/check-ucarp-procs.sh

```diff
@@ -6,7 +6,12 @@ for IFACE in $(ifquery --list); do
       echo "no ucarp process is running for IP ${VIP}"
       exit 2
     fi
-  done
+    for HOSTNAME in `hostname -s`; do
```

Comment:
However you are setting a variable,  you're just setting it as part of a loop statement for something that [should] only ever give a single item response.   Having a loop here could confuse a reader as to the purpose of the test.